### PR TITLE
refactor: disable max params for functions

### PR DESCRIPTION
**Pull request by LucasFalkowsky**
max-params is turned off

### CODE REVIEW

File: tests/base/function-param-test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const testFunction = (a, b, c, d) => `${a} ${b} ${c} ${d}`;
+
+console.log(testFunction('Karl', 'is', 'my', 'friend'));
```
Comment (timidak) [inline tests/base/function-param-test.js:3]: I guess you can remove the `console.log` and just invoke the function

File: tests/base/base.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+describe('base config', () => {
+  it('should allow more then 3 params passed to function', async () => {
+    // arrange
+    const linter = new ESLint();
+    const fileName = 'tests/base/function-param-test.js';
+
+    // act
+    const [result] = await linter.lintFiles([fileName]);
+
+    expect(result.errorCount).toBe(0);
```
Comment (timidak) [inline tests/base/base.test.js:13]: The rule warned us and didn't give an error, so we should check for the rule identifier in the messages.

File: tests/base/base.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+describe('base config', () => {
+  it('should allow more then 3 params passed to function', async () => {
+    // arrange
+    const linter = new ESLint();
+    const fileName = 'tests/base/function-param-test.js';
+
+    // act
+    const [result] = await linter.lintFiles([fileName]);
+
```
Comment (timidak) [inline tests/base/base.test.js:12]: add an `// assert` here or remove we other ones.

**Review by timidak — CHANGES_REQUESTED** [review]
Some small changes to be made. 😄

File: tests/base/base.test.js (hunk 2 above)
Comment (LucasFalkowsky) [reply tests/base/base.test.js:13]: makes sense!

File: tests/base/base.test.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    // assert
+    expect(result.warningCount).toBe(0);
```
Comment (timidak) [inline tests/base/base.test.js:14]: I might be wrong, but this test fails I guess.

**Review by timidak — CHANGES_REQUESTED** [review]
Some changes to be made

**Review by timidak — APPROVED** [review]
LGTM 🎉

**Pull request merged**